ui/src/app/components/plans/test-plan-editor/test-plan-suite-machine-selection.component.html

```diff
@@ -86,6 +94,8 @@
           (onCreateMachine)="selectedTestSuites.length && handleCreateMachine()"
           (onMapMachine)="mapTestSuitesToMachine($event, this.selectedTestSuites, false)"
         ></app-create-machine-button>
+        <button class="theme-btn-primary ml-15" (click)="addSuites()"
```

Comment:
Angular is giving an error. because, button is neither void element nor custom tag. so angular is throwing a parsing error.


github discussion: https://github.com/angular/angular/issues/5563